buildSrc/src/main/kotlin/elastic-otel.library-packaging-conventions.gradle.kts

```diff
@@ -11,7 +11,6 @@ tasks {
 
       include("LICENSE")
       include("NOTICE")
-      include("licenses/**")
```

Comment:
Our library project are *not* fat-jars: Their dependencies are not shipped with them, but listed in the pom.
Therefore I think it is wrong to include the licenses of those dependencies within the library jars.